src/engine/helper/ec_local.h

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright 2001-2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2002, Oracle and/or its affiliates. All rights reserved
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdlib.h>
+#include <openssl/obj_mac.h>
+#include <openssl/ec.h>
+#include <openssl/bn.h>
+
+/* Note: Although these structures are internal to OpenSSL, they are defined
```

Comment:
multi-line comments should be as follow:

     /*
      * Multi-line
      * comment.
      */

Reply:
Yes, Thanks.